CK3toEU4/Source/EU4World/EU4World.cpp

```diff
@@ -663,16 +663,37 @@ void EU4::World::linkProvincesToCountries()
 	// Super Mongolia?
 	if (countries.count("MGE") && !countries.find("MGE")->second->getProvinces().empty())
 	{
-		// Move all Mongolia provinces under Mongol Empire
-		for (const auto& province: provinces)
+		bool isSource = false;
+		for (auto province: countries.find("KHA")->second->getProvinces())
```

Comment:
KHA may not exist in which case this will crash.

Reply:
KHA always exists, because it always holds lands in vanilla.